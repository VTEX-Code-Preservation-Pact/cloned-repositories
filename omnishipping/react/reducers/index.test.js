import index from './index.js'
import { createStore } from 'redux'

describe('Combined Reducers', () => {
  it('should load combined reducers', () => {
    const combinedReducers = createStore(index)
    expect(combinedReducers.getState()).toMatchSnapshot()
  })
})
