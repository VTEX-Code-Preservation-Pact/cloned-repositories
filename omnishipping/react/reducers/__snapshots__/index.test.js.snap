// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Combined Reducers should load combined reducers 1`] = `
Object {
  "accountInfo": Object {
    "acceptSearchKeys": Array [
      "postalCode",
    ],
    "accountName": undefined,
    "geolocation": true,
    "googleMapsKey": undefined,
    "loadingAccountInfo": false,
  },
  "addressForm": Object {
    "addresses": Object {},
    "alreadyCompletedReceiverName": false,
    "queuedValidations": Array [],
    "receiverName": null,
    "valid": null,
  },
  "componentActivity": Object {
    "activeDeliveryChannel": "delivery",
    "editAddressActive": false,
    "hasPickup": false,
    "isAddressActive": false,
    "isAddressListActive": false,
    "isEditingOmnishipping": false,
    "isOmniShippingCompleted": false,
    "isOmniShippingOpen": false,
    "isSplitActive": false,
    "newAddressActive": false,
    "triedCompleteOmnishipping": false,
  },
  "delivery": Object {
    "CHEAPEST": Array [],
    "COMBINED": Array [],
    "FASTEST": Array [],
    "activeDeliveryOption": "CHEAPEST",
    "activeScheduledDelivery": null,
    "isScheduledDeliveryActive": false,
    "logisticsInfo": Array [],
    "optionsDetails": null,
    "selectedDeliveryPackageSlider": 0,
    "shouldDisableToggle": false,
    "shouldUpdateLeanOptionsState": false,
    "shouldUpdateUi": false,
    "simulateRequested": false,
  },
  "orderForm": Object {},
  "pickup": Object {
    "activePickupPoint": null,
    "activeSellerId": "",
    "askForGeolocation": null,
    "hasGeolocation": false,
    "isModalActive": false,
    "isPickupDetailsActive": false,
    "isSearching": false,
    "pickupOptions": Array [],
    "pickupPoint": null,
  },
}
`;
